cmd/minikube/cmd/start.go

```diff
@@ -303,6 +303,7 @@ func clusterUp(config *cluster.MachineConfig) {
 	// Set default value for host config and data
 	viper.Set(hostConfigDir, viper.GetString(hostConfigDir))
 	viper.Set(hostDataDir, viper.GetString(hostDataDir))
+	viper.Set(hostVolumesDir, viper.GetString(hostVolumesDir))
```

Comment:
here we would have to add the validation. The VM is up and running. We can check and create the directory.